performancecopilot/pcp
# qa

Changes committed to git@github.com:kmcdonell/pcp.git 20200126

Ken McDonell (8):
      qa/581: add another expected disk name to the filter
      qa/243: tweak logic
      qa/common.filter: add netcheck pmda to _filter_optional_pmda_instances()
      qa/admin/other-packages/unavailable: no bpftrace for CentOS 7.7
      qa/950: small changes
      scripts/pcp-push: add blank line after PR title
      qa/1160: filter out Python unicode u'...' babble
      qa/1161: filter out Python unicode u'...' babble

 qa/1160                             |    1 +
 qa/1161                             |    1 +
 qa/243                              |   36 ++++++++++++++++++++++++++++--------
 qa/243.out.ipv6                     |    5 ++---
 qa/243.out.nonipv6                  |    5 ++---
 qa/581                              |    1 +
 qa/950                              |   11 ++++++++++-
 qa/admin/other-packages/unavailable |    2 +-
 qa/common.filter                    |    1 +
 scripts/pcp-push                    |    1 +
 10 files changed, 48 insertions(+), 16 deletions(-)

Details ...

commit 4860061faeb783afc92bd9d25f6a042ccc49d088
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 21:35:45 2020 +1100

    qa/1161: filter out Python unicode u'...' babble
    
    Was failing on CentOS 7.7 (vm04).

commit dfc499489cf0c7630259c66e31933a2ebf8cd2fc
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 21:32:02 2020 +1100

    qa/1160: filter out Python unicode u'...' babble
    
    Was failing on CentOS 7.7 (vm04).

commit 83d8ba361a0547d8fd4d661f329a0031ae7bace5
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 21:24:20 2020 +1100

    scripts/pcp-push: add blank line after PR title

commit 43f4cf7723d631c0ba6a65ecdaaa678db23866fc
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 14:42:32 2020 +1100

    qa/950: small changes
    
    - avoid race between avahi_service and avahi_verify (the service can
      be "started" from sysctl but the socket is not yet created) ... add
      an arbitrary 5 sec sleep
    - filter out timeout errors when trying to contact other (remote) pmproxy
      services
    
    Without this change, the test was always failing on CentOS 7.7 (vm04).

commit e83404202f9208c6f5fb7ff37b00d71d894c1c80
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 14:31:15 2020 +1100

    qa/admin/other-packages/unavailable: no bpftrace for CentOS 7.7
    
    Assume same holds for earlier 7.x releases.

commit c40baafcd83709602afc4dea5463e192e6ba4a01
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 14:24:36 2020 +1100

    qa/common.filter: add netcheck pmda to _filter_optional_pmda_instances()
    
    Was causing qa/650 failures on CentOS 7.7 (vm04) where the netcheck
    PMDA was installed (not by default, but probably caused by an unrelated
    QA failure in the netcheck PMDA tests).

commit 0adf67ce92eee694d72757c1e854e42617901870
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 14:10:28 2020 +1100

    qa/243: tweak logic
    
    1. be a little more patient waiting for the first pmcd.log to appear
    2. use a different log file for the second pmcd ... makes error
       checking and diff'ing logic clearer.
    
    Apparently avoids non-deterministic failures on CentOS 7.7 (vm04).

commit f90192c8acfe7ba32c261458ac421bf5223ba216
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sun Jan 26 14:09:03 2020 +1100

    qa/581: add another expected disk name to the filter
    
    da[0-9] this time ... from FreeBSD 11 (vm10).
